### DISCUSSION TITLE
Extending Rocket.rs capabilities

### DISCUSSION THREAD

**Discussion by davidjeba** [rwf2/Rocket#2430] [Ideas]
Hi,

i recently tried https://github.com/typesense/typesense-rust
i think Typesense requires to be in Rocket.rs
and
Rocket.rs will be benefitted by typesense

**Comment by nathanielford** [comment]
It's possible that your English isn't strong enough to make clear what you want, but if (as I think is the case?) you'd like Rocket to include Typesense, you should be clear what benefits Rocket would get from it, and where it would be utilized. That said, you shouldn't hold your breath on this - it's not clear from Typesense's what it's use case is, or that it's production-ready.
